test/datasets/datasets_test.py

```diff
@@ -1,19 +1,15 @@
+from torchaudio.datasets.cmuarctic import CMUARCTIC
```

Comment:
Please keep the changes to minimum, and do not change lines for the sake of formatting. (it's okay to format the lines you are touching)

Running formatter is nice but it will disturb `git blame` later and it becomes hard to reason about changes in the future.